Source/ActivityPub.Types/Conversion/ASTypeInfoCache.cs

```diff
@@ -59,6 +59,12 @@ public interface IASTypeInfoCache
 /// </summary>
 public class ASTypeInfoCache : IASTypeInfoCache
 {
+    private static readonly Lazy<ASTypeInfoCache> Lazy = new(() =>
+    {
+        var cache = new ASTypeInfoCache();
+        cache.RegisterAllAssemblies();
```

Comment:
I've thought about this a lot, and I think it makes more sense to move this call to the constructor. Then we can remove the two public "Register" methods, which makes this class immutable.